openrewrite/package.json

```diff
@@ -33,6 +33,7 @@
     "@types/diff": "^5.2.2",
     "@types/jest": "^29.5.13",
     "@types/uuid": "^10.0.0",
+    "@openrewrite/rewrite-remote" : "~0.3.0",
```

Comment:
We can try to add this, but when you also need to change the LST model, I think there is no way around `npm link`, which will remove this dependency again.